cdap-watchdog/src/main/java/co/cask/cdap/metrics/collect/AggregatedMetricsCollectionService.java

```diff
@@ -110,8 +110,8 @@ protected Scheduler scheduler() {
   }
 
   @Override
-  public final MetricsCollector getCollector(final MetricsScope scope, final Map<String, String> tags) {
-    return collectors.getUnchecked(new CollectorKey(scope, tags));
+  public final MetricsCollector getCollector(final Map<String, String> tags) {
+    return collectors.getUnchecked(new CollectorKey(MetricsScope.SYSTEM, tags));
```

Comment:
yup - that is bridging changes of this step to this one: https://github.com/caskdata/cdap/pull/991

I.e. fixed in 2nd step